java/src/main/java/ai/rapids/cudf/RmmAllocationMode.java

```diff
@@ -28,4 +28,8 @@
    * Use cudaMallocManaged rather than cudaMalloc
    */
   public static final int CUDA_MANAGED_MEMORY = 0x00000002;
+  /**
+   * Use arena suballocation strategy
+   */
+  public static final int ARENA = 0x00000003;
```

Comment:
This is a bitset not an enum.  Before this change, 0x3 meant `POOL | CUDA_MANAGED_MEMORY` meaning we want to use the pool allocator backed by managed memory.  I would assume similarly that we would want the ability to specify the arena allocator backed by either normal CUDA memory or managed memory.

Reply:
Fixed.

Reply:
oh nice catch